vuln/npm/405.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": 405,
+  "title": "Cross-site Scripting (XSS) - Stored",
+  "overview": "public allows to embed HTML in file names, which (in certain conditions) might lead to execute malicious JavaScript.",
+  "created_at": "2018-04-15",
```

Comment:
all the created_at and updated_at dates through-out the vulns should be ISO 8601 with the recent contribution from @greysteil

Reply:
I think YYYY-MM-DD is a valid ISO 8601 date 🤔 (see 5.2.1.1 in http://lists.ebxml.org/archives/ebxml-core/200104/pdf00005.pdf )

Reply:
It is. I could look into adding a validation that minute-precision is specified, or something like that?

Reply:
I think precision smaller than day is a bit overkill for such records. What is your take here?

Reply:
People might be watching the `updated_at` and want greater precision? I think I agree though - that's going to be pretty rare, and if they are then they'd be better of storing the commit SHA of the repo. Shall we change them all to dates? I'll put in a PR.

Reply:
yeah it's valid as a partial, however all the vulns are formatted with the full timestamp

Reply:
Right SHAs should do the trick in times of need. Let's just see if @lirantal has an opinion here but I'd be 👍 for a PR.

Reply:
I'm good 👍 
we can iterate on further enhancements as required